webapp/package.json

```diff
@@ -33,5 +34,10 @@
     "qs": "6.9.4",
     "Select2": "github:select2/select2#3.5.4",
     "tablesorter": "2.31.3"
+  },
+  "scripts": {
+    "build": "rm -rf static/* && npm run copy:images && node esbuild && npm run build:collectstatic",
+    "build:collectstatic": "TOOL_ENV=True ./manage.py collectstatic --noinput --ignore css --ignore img",
```

Comment:
Note: the two `--ignore` flags also ignore css and images for Django admin.  From my research, the glob-style flags are notoriously brittle/unintuitive – I was unable to set up `crashstats/**/*.css` and `crashstats/**/img/`.  Once JS files are included in the new build process, we can more easily do `--ignore crashstats` so that collectstatic's only concern is admin.

We should remove both ignore flags before merge, but I left them here and in `test.sh` to illustrate/confirm that ESBuild is in fact building/responsible for CSS/img files.